cirq-core/cirq/transformers/analytical_decompositions/two_qubit_to_cz.py

```diff
@@ -31,13 +32,25 @@
     import cirq
 
 
+@_compat.deprecated(fix='Please use two_qubit_matrix_to_cz_operations', deadline='v0.15')
 def two_qubit_matrix_to_operations(
     q0: 'cirq.Qid',
     q1: 'cirq.Qid',
     mat: np.ndarray,
     allow_partial_czs: bool,
     atol: float = 1e-8,
     clean_operations: bool = True,
+) -> List[ops.Operation]:
+    return two_qubit_matrix_to_cz_operations(q0, q1, mat, allow_partial_czs, atol, clean_operations)
```

Comment:
nit: Don't remove the docstring on the original deprecated method, here and below.

Reply:
Done.